hrjobcontract/tests/phpunit/CRM/Hrjobcontract/Import/Parser/ApiTest.php

```diff
@@ -47,6 +53,22 @@ function setUp() {
     ];
   }
 
+  function createPayScale() {
```

Comment:
As I understand, we don't need teardown methods because the test database gets re-installed and upgraded at the beginning of each test, to start with a clean slate, so to speak.